jkube-kit/generator/api/src/main/java/org/eclipse/jkube/generator/api/support/BaseGenerator.java

```diff
@@ -78,6 +78,7 @@ enum Config implements Configs.Config {
 
         // Base image mode (only relevant for OpenShift)
         FROM_MODE("fromMode", null),
+        BUILDPACKS_BUILDER_IMAGE("buildpacksBuilderImage", null),
```

Comment:
OK, now this starts to make more sense. Why isn't the default builder image resolved from the configuration?

Reply:
OK, I recall there was a similar situation for other configuration flags.
I can't recall how those got fixed.

Reply:
Let's move on with this, and we can check at another moment